web/src/components/__tests__/SearchFeedback.spec.tsx

```diff
@@ -50,4 +51,19 @@ describe('SearchFeedback', () => {
     )
     expect(getByText('feedback:send')).toBeTruthy()
   })
+
+  it('should not allow sending search feedback if query term is removed', async () => {
+    const { getByText, rerender } = renderWithTheme(
+      <SearchFeedback cityCode={cityCode} languageCode={languageCode} query='ab' noResults />,
+    )
+    expect(getByText('feedback:send')).toBeTruthy()
+
+    // the query is controlled in the parent of SearchFeedback, so we need to update the props
+    rerender(
+      <ThemeProvider theme={theme}>
```

Comment:
Can't we use `renderWithTheme` here as well?

Reply:
Unfortunately not, we need to use the `rerender` that the earlier `renderWithTheme` returns. Otherwise the `SearchFeedback` gets rendered a second time with the new props, as opposed to rerendering the first one with new props.

Reply:
Ahh sorry I missed that. Nvm :)